payment/key.go

```diff
@@ -115,6 +63,7 @@ func GenerateSecret() (secret string, nonce string, err error) {
 	if err != nil {
 		return "", "", err
 	}
+	encryptedBytes, nonceBytes, err := cryptography.EncryptMessage([]byte(unencryptedsecret))
```

Comment:
nit: should be unencryptedSecret if i am understanding https://github.com/unknwon/go-code-convention/blob/master/en-US/naming_rules.md#variables correctly

Reply:
Nice catch! https://github.com/brave-intl/bat-go/pull/347/commits/40a3ace8a600a38fa6bbf9391eeb481993501595